plugins/vpp/l3plugin/vppcalls/vpp2005/vrrp_vppcalls.go

```diff
@@ -31,6 +31,9 @@ func (h *VrrpVppHandler) vppAddDelVrrp(entry *l3.VRRPEntry, isAdd uint8) error {
 	var addrs []ip_types.Address
 	var isIpv6 bool
 	for idx, addr := range entry.IpAddresses {
+		if addr == "localhost" {
```

Comment:
Why handling this specific case? This should actually never happen because it won't pass through Validate: https://github.com/NotSoFancyName/vpp-agent/blob/vrrp-support/plugins/vpp/l3plugin/descriptor/vrrp.go#L115-L118

Reply:
https://play.golang.org/p/Lv9iWoaTuey